NoteFind/Alerts/ErrorAlerts.m

```diff
@@ -0,0 +1,111 @@
+//
+//  ErrorAlerts.m
+//  NoteFind
+//
+//  Created by Khloe Wright on 7/22/22.
+//
+
+#import "ErrorAlerts.h"
+
+@implementation ErrorAlerts
+
```

Comment:
Assuming you just implement the `showAlertForErrorType:(ErrorAlertType)errorType` API, you would just need a single method in this file with a `switch` statement that handles all the cases in your error enum to display the proper message / logic to the user.